### ISSUE TITLE
Add to Homebrew Core

### ISSUE THREAD

**Issue by ritzmann** [walles/homebrew-johan#6]
Nice, I love your tools and was looking at packaging them for Homebrew a few weeks ago and finally wanted to start working on it today and there I find that it is already done! 😸

But I guess you would still need some third party, i.e. me, to submit the formulas to Homebrew Core?

As far as I understand, I would just need to fork Homebrew/homebrew-core, copy your formulas (one at a time) and create a pull request for each formula?

**Comment by walles** [comment]
Glad you like my stuff :)

Yes, a third party volunteer would be needed, thanks for stepping up!

Here's Homebrew's entry point for how to PR new formulas:
<https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md#to-add-a-new-formula-for-foo-version-234-from-url>

**Comment by ritzmann** [comment]
Yep, I'll send up a trial balloon for moar and let's see how that goes.

**Comment by ritzmann** [comment]
Alright, we'll see how that goes: https://github.com/Homebrew/homebrew-core/pull/79087

**Comment by ritzmann** [comment]
That went really quick. The moar formula was merged last night. I will continue with the rest in the next few days.

**Comment by ritzmann** [comment]
Here goes px: https://github.com/Homebrew/homebrew-core/pull/79399
I submitted with the git download strategy. We'll find out what the reviewers say.

**Comment by ritzmann** [comment]
px was merged!

The git download was no problem. Turns out that Homebrew formulas don't use the `using: :git` tag and that's why my query did not find anything.

**Comment by ritzmann** [comment]
Feel free to ping me if/when riff has reached the Homebrew popularity threshold.

**Comment by walles** [comment]
Cool, thanks a lot Fabian!

I have updated the official install instructions of both `px` and `moar` to refer to homebrew core for installs, and their respective release instructions to add new releases there.

**Comment by walles** [comment]
`riff` specific conversation here: <https://github.com/walles/riff/issues/20>
